test/conformance/tests/sriov_operator.go

```diff
@@ -913,9 +913,72 @@ var _ = Describe("[sriov] operator", func() {
 				createUnschedulableTestPod(testNode, []string{sriovNetworkName}, resourceName)
 			})
 		})
+
+		Context("SRIOV and macvlan", func() {
+			// 25834
+			It("Should be able to create a pod with both sriov and macvlan interfaces", func() {
+
+				resourceName := "sriovnic"
+				sriovNetworkName := "sriovnetwork"
+				testNode := sriovInfos.Nodes[0]
```

Comment:
should we check the len of the nodes or we already do it in the before each?

Reply:
Checked here:
https://github.com/openshift/sriov-network-operator/blob/master/test/util/cluster/cluster.go#L56